spec/models/fee_calculator_spec.rb

```diff
@@ -59,6 +59,50 @@
       it { is_expected.to eq(400) }
     end
 
+    context '12時間以内1200円' do
+      let(:attr) do
+        {
+         'type' => 'within',
+         'minute' => 720,
+         'fee' => 1200,
+         'repeat' => false,
+         'within' => {
+                      'type' => 'basic',
+                      'per_minute' => 60,
+                      'fee' => fee,
+                     }
+        }
+      end
+
+      context '1時間3000円' do
+        let(:fee) { 3000 }
+        it { is_expected.to eq(1200) }
+      end
+
+      context '1時間100円' do
+        let(:fee) { 100 }
+        it { is_expected.to eq(100) }
+      end
+    end
+
+    context '12時間以内1200円' do
+      let(:attr) do
+        {
+         'type' => 'within',
+         'minute' => 720,
+         'fee' => 1200,
+         'repeat' => false,
+         'within' => {
+                      'type' => 'basic',
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.